scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -309,6 +309,7 @@ def pytest_run(self, timeout):
 
     def generate_command(self):
         config = self.instance.testsuite.harness_config
+        handler: Handler = self.instance.handler
         pytest_root = config.get('pytest_root', ['pytest']) if config else ['pytest']
         pytest_args = config.get('pytest_args', []) if config else []
```

Comment:
Could you change name of this variable to `pytest_args_yaml` to distinguish them from this one which came from CLI?